pages/api/book/[isbn].ts

```diff
@@ -2,7 +2,8 @@ import type { NextApiRequest, NextApiResponse } from 'next';
 import { HttpMethod } from "@lib/http-method";
 import { StatusCodes } from "http-status-codes";
 import { getBookWithPosts } from "../../../lib/services/book";
-import { isValidISBN } from "../../../lib/helpers/backend/valid-isbn";
+
+var ISBN = require('isbn').ISBN;
```

Comment:
Hmm this package doesn't even have TypeScript types, I think the package I gave you might not have been the one I meant. I still think we probably don't even need ISBN validation since it's just a GET operation though.

Reply:
(also see comment on old PR)

Reply:
Will be removed.